stylesheets/layout/_header.scss

```diff
@@ -1,51 +1,56 @@
-// -----------------------------------------------------------------------------
-// This file contains all styles related to the header of the site/application.
-// -----------------------------------------------------------------------------
 .header {
-  display: flex;
-  justify-content: space-between;
+  @include display;
   margin-top: 44px;
 }
-.logotype{
+
+.logotype {
   display: flex;
-  img, h1{
+  
+  .logotype-image, .logotype-header {
     align-self: center;
     margin-right: 13px;
-
+    @include typography-weight700 (30px, 34px);
+    color: $logo-color;
   }
+
 }
```

Comment:
```
.logotype {
   display: flex;

   &-image,
   &-header {
     align-self: center;
     margin-right: 13px;
     @include typography-weight700 (30px, 34px);
     color: $logo-color;
   }
 }
```

Use & to avoid selector repeating.
Each new selector should on a new line.